pages/mercury_tour_page.py

```diff
@@ -0,0 +1,76 @@
+import time
+from selenium.webdriver.support.select import Select
+
+
+class BasePage(object):
+
+    def __init__(self, driver):
+        # instantiate driver
+        self.driver = driver
+
+
+class MainWelcomePage(BasePage):
+
+    def title_matches(self):
+        # Find out Header
+        return "Welcome: Mercury Tours" in self.driver.title
+
+    def login_cred(self):
+        # Give login cred and click submit
+        user_name = self.driver.find_element_by_name('userName')
+        user_name.send_keys("admin")
```

Comment:
username and password should be sent from the function you are calling this method, try something like:
```
def login_cred(self, username, password):
    user_name = self.driver.find_element_by_name('userName')       
    user_name.send_keys(username)
```

When you will call this function:
```
login_cred('admin', 'password_here')
```

Reply:
Done